dataprofiler/profilers/profiler_utils.py

```diff
@@ -177,6 +177,34 @@ def partition(data: list, chunk_size: int) -> Generator[list, None, Any]:
         yield data[idx : idx + chunk_size]
 
 
+def auto_multiprocess_toggle(
+    data: DataFrame,
+    num_rows_threshold: int = 750000,
+    num_cols_threshold: int = 20,
+) -> bool:
+    """
+    Automate multiprocessing toggle depending on dataset sizes.
+
+    :param data: a dataset
+    :type data: pandas.DataFrame
+    :param num_rows_threshold: threshold for number of rows over
+        which options.multiprocess is enabled
```

Comment:
not technically true... its the threshold at which if `options.multiprocess == True` we will check dataset size for multiprocess?